iceoryx_examples/request_response_basic/client_cxx_api_event.cpp

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) 2022 by Apex.AI Inc. All rights reserved.
```

Comment:
I would rename this example either into `client_cxx_listener` or `client_cxx_push_notification` or something similar. I think `event` is not so expressive.